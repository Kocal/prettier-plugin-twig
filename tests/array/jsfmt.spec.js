run_spec(__dirname, ["twig"]);
run_spec(__dirname, ["twig"], { printWidth: 80 });
