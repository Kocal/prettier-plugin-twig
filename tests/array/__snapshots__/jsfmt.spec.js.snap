// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`array.twig 1`] = `
Object {
  Symbol(jest-snapshot-serializer-raw): "====================================options=====================================
parsers: [\\"twig\\"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
{% set foo = [1, 2, 3] %}
{% set foo = [ 1,   2,   3  ] %}
{% set foo = [ 'an', 'array',
    'of', 'string'   ] %}

{%  set foo = ['aVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryLongArray', 'aVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryLongArray', 'aVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryLongArray', 'aVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryLongArray']   %}

{% set foo = [
    'a',
    'b',
    'c',
    'd',
    'e',
    'f',
    'g',
] %}

{% set foo = ['aVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryLongValue'] %}

{% set foo = [1,2,3,\\"string\\",\\"bar\\",{foo:'bar'} ] %}

=====================================output=====================================
{% set foo = [1, 2, 3] %}
{% set foo = [1, 2, 3] %}
{% set foo = [\\"an\\", \\"array\\", \\"of\\", \\"string\\"] %}

{% set foo = [
    \\"aVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryLongArray\\",
    \\"aVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryLongArray\\",
    \\"aVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryLongArray\\",
    \\"aVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryLongArray\\"
] %}

{% set foo = [\\"a\\", \\"b\\", \\"c\\", \\"d\\", \\"e\\", \\"f\\", \\"g\\"] %}

{% set foo = [
    \\"aVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryLongValue\\"
] %}

{% set foo = [1, 2, 3, \\"string\\", \\"bar\\", { foo: \\"bar\\" }] %}

================================================================================",
}
`;

exports[`array.twig 2`] = `
Object {
  Symbol(jest-snapshot-serializer-raw): "====================================options=====================================
parsers: [\\"twig\\"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
{% set foo = [1, 2, 3] %}
{% set foo = [ 1,   2,   3  ] %}
{% set foo = [ 'an', 'array',
    'of', 'string'   ] %}

{%  set foo = ['aVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryLongArray', 'aVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryLongArray', 'aVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryLongArray', 'aVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryLongArray']   %}

{% set foo = [
    'a',
    'b',
    'c',
    'd',
    'e',
    'f',
    'g',
] %}

{% set foo = ['aVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryLongValue'] %}

{% set foo = [1,2,3,\\"string\\",\\"bar\\",{foo:'bar'} ] %}

=====================================output=====================================
{% set foo = [1, 2, 3] %}
{% set foo = [1, 2, 3] %}
{% set foo = [\\"an\\", \\"array\\", \\"of\\", \\"string\\"] %}

{% set foo = [
    \\"aVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryLongArray\\",
    \\"aVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryLongArray\\",
    \\"aVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryLongArray\\",
    \\"aVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryLongArray\\"
] %}

{% set foo = [\\"a\\", \\"b\\", \\"c\\", \\"d\\", \\"e\\", \\"f\\", \\"g\\"] %}

{% set foo = [
    \\"aVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryVeryLongValue\\"
] %}

{% set foo = [1, 2, 3, \\"string\\", \\"bar\\", { foo: \\"bar\\" }] %}

================================================================================",
}
`;
